src/components/FormRenderer/index.tsx

```diff
@@ -46,6 +46,8 @@ export interface FormRendererProps {
     validatorMapper?: ValidatorMapper;
 }
 
+export const SubmittingContext = React.createContext<boolean | undefined>(false);
```

Comment:
Let us call it `FormRendererContext` and then make it an object to keep the interface open.